Kickstarter-iOS/Views/Controllers/SelectCurrencyViewController.swift

```diff
@@ -0,0 +1,79 @@
+import Library
+import Prelude
+import Prelude_UIKit
+import UIKit
+
+private let reuseIdentifier = "CurrencySelectionCell"
```

Comment:
Not a suggestion for this PR but we should start extracting these to a singular place (my PRs are going the same direction :-)

Reply:
Yeah let's chat about it!